src/components/bulk-email-tool/TextEditor.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Editor } from '@tinymce/tinymce-react';
+import PropTypes from 'prop-types';
+import 'tinymce';
+
+import 'tinymce/icons/default';
+import 'tinymce/themes/silver';
+import 'tinymce/skins/ui/oxide/skin.css';
+import 'tinymce/plugins/advlist';
+import 'tinymce/plugins/code';
+import 'tinymce/plugins/emoticons';
+import 'tinymce/plugins/emoticons/js/emojis';
+import 'tinymce/plugins/link';
+import 'tinymce/plugins/lists';
+import 'tinymce/plugins/table';
+import 'tinymce-language-selector';
+
+import contentUiCss from 'tinymce/skins/ui/oxide/content.css';
+import contentCss from 'tinymce/skins/content/default/content.css';
+
+export default function TextEditor(props) {
+  const {
+    onChange, onKeyUp, onInit, disabled,
+  } = props;
+
+  return (
+    <Editor
+      initialValue=""
+      init={{
+        selector: 'textarea#editor',
+        height: 600,
+        plugins: 'advlist code emoticons link lists table',
+        toolbar: 'bold italic | bullist numlist | link emoticons',
```

Comment:
Yeah I left it slim for now, but a ton of plugins are available -- plus we can write any we need.

Reply:
Do you think we should do the research to compare what we have now and get parity, or do you see this as something we do later? If it is later, should we create a ticket so we don't miss it?

Reply:
I'm not sure we need to worry about this now. I'll create a subtask to check the plugins and make it a part of this ticket.